default.nix

```diff
@@ -32,12 +32,15 @@ let
   };
   pkgs = import sources.nixpkgs { inherit system config; };
   sources = localLib.sources // {
-    cardano-wallet = pkgs.runCommand "cardano-wallet" {} ''
-      cp -r ${localLib.sources.cardano-wallet} $out
-      chmod -R +w $out
-      cd $out
-      patch -p1 -i ${./nix/cardano-wallet--enable-aarch64-darwin.patch}
-    '';
+    cardano-wallet =
+      if target != "aarch64-darwin"
```

Comment:
what's wrong with aarch64-darwin from cardano-wallet?

Reply:
Ummm, it’s not being built in the original repo, since we don’t have official `aarch64-darwin` machines yet :(

We patch it here to enable those builds, which work fine, just are not official